platform/v05/exit.go

```diff
@@ -0,0 +1,31 @@
+package v05
+
+var exitCodes = map[string]int{
```

Comment:
Do we really need an `exit.go` file?
Can't we just have one file that implements the Platform interface for version 0.5 (and the same for version 0.6)?
Don't have a strong opinion on this. Just curios.

Reply:
Summarizing other discussion, we plan to extend the platform interface to cover other platform related logic so having a separate file probably makes sense.